ui/editor/src/view.js

```diff
@@ -154,6 +170,16 @@ module.exports = function(ctrl) {
   var fen = ctrl.computeFen();
   var color = ctrl.chessground.data.orientation;
   var opposite = color === 'white' ? 'black' : 'white';
+  var sparePieceSelected = ctrl.vm.selected();
+  var selectedParts = sparePieceSelected.split(' ');
+  var cursorName = selectedParts[0] + ((selectedParts.length >= 2) ? '-' + selectedParts[1] : '');
+
+  ctrl.chessground.sparePieceSelected = sparePieceSelected;
+
+  // http://www.cursors-4u.com
+  ctrl.chessground.cursor = (cursorName === 'pointer') ?
```

Comment:
let's not mess with chessground for that. Instead, apply the cursor to the editor element that wraps chessground.